frontend/src/pages/Organizations/Organizations.tsx

```diff
@@ -87,6 +113,21 @@ export const Organizations: React.FC = () => {
           </>
         )}
       </div>
+      <Dialog open={dialogOpen} onClose={handleClose}>
+        <DialogTitle id="alert-dialog-title">
+          {'Organization Upload Feedback:'}
+        </DialogTitle>
+        <DialogContent>
+          <div> Successful uploads: {successCount}</div>
+          <div> Failed uploads: {failureCount}</div>
```

Comment:
We should just say "XX organizations uploaded successfully" if all uploads were successful. Only if the number of failed uploads is greater than zero should we also add "XX organizations' uploads failed."

Reply:
Updated